### DISCUSSION TITLE
Dropdowns needs a way to filter items

### DISCUSSION THREAD

**Discussion by CBelmonte** [RocketCommunicationsInc/astro#870] [Feature Requests]
When large lists exist, a way to filter items inside dropdowns become necessary.

**Comment by CBelmonte** [comment]
Something like this:
https://semantic-ui.com/modules/dropdown.html#search-dropdown

**Comment by micahjones13** [comment]
Hey @CBelmonte just to clarify, are you using rux-select as the dropdown in this instance?

**Comment by QGerrit** [comment]
I was looking for something similar too -- Currently using a `rux-select`, but I have ~1000 items in it, so a search bar for it would be really nice. Would be nice to not write one myself :)
